packages/gatsby/package.json

```diff
@@ -50,11 +50,12 @@
     "eslint-config-react-app": "^3.0.0",
     "eslint-loader": "^2.1.0",
     "eslint-plugin-flowtype": "^2.46.1",
-    "eslint-plugin-graphql": "^2.0.0",
+    "eslint-plugin-graphql": "^3.0.3",
     "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.8.2",
     "express": "^4.16.3",
+    "express-graphql": "^0.7.1",
     "express-graphql": "^0.6.12",
```

Comment:
Have you noticed that the `express-graphql` is set twice in the dependencies? It's not currently using the version `^0.7.1` you have set, but instead the `^0.6.12`.

Reply:
didn't notice will fix that..